content/en/docs/setup/install/kubernetes/index.md

```diff
@@ -83,7 +83,7 @@ All newly deployed workloads will have Istio sidecars installed.
 Run the following command to install this variant:
 
 {{< text bash >}}
-$ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
+$ helm template install/kubernetes/helm/istio --name istio --namespace istio-system --values install/kubernetes/helm/istio/values-istio-demo.yaml --set global.controlPlaneSecurityEnabled=true --set global.mtls.enabled=true | kubectl apply -f -
```

Comment:
You can't use helm in this guide. These are the quick start instructions for people that do NOT have or want to install helm.

Reply:
Thanks @frankbu I just noticed that `This guide installs Istio’s built-in demo configuration profile using basic Kubernetes commands without needing to download or install Helm. ` since we have removed `istio-demo-auth.yaml`, how to install it without `helm`?
Any suggestions? @howardjohn

Reply:
just get rid of all the auth mode here - we will just have the single demo